mkdocs.yml

```diff
@@ -0,0 +1,27 @@
+site_name: GeoTrellis
+site_author: GeoTrellis contributers
+repo_url: https://github.com/rossbernet/geotrellis/tree/master/docs
```

Comment:
We can switch this to the real repo URL, since we do all our debugging/prototyping locally with `mkdocs`.